src/App/Header/Connect.svelte

```diff
@@ -0,0 +1,236 @@
+<script lang="ts">
+  import debounce from "lodash/debounce";
+
+  import { closeFocused } from "@app/components/Floating.svelte";
+  import { sessionStore, disconnect } from "@app/lib/session";
+  import { toClipboard } from "@app/lib/utils";
+
+  import Avatar from "@app/components/Comment/Avatar.svelte";
+  import Button from "@app/components/Button.svelte";
+  import Floating from "@app/components/Floating.svelte";
+  import Icon from "@app/components/Icon.svelte";
+
+  function formatId(id: string) {
```

Comment:
Could we maybe use the `formatRadicleId` from `utils.ts`? Or did you want to have a shorter id?
https://github.com/radicle-dev/radicle-interface/blob/2c5830a076fe08dc01c87476263f11ac9f396c9f/src/lib/utils.ts#L49

Reply:
It won't work because we're doing `assert(isRadicleId(id));` for some reason in that function and this id doesn't include `rad:` so the assert would fail. But even if we remove that, I still don't think it looks good in the button:
<img width="301" alt="Screenshot 2023-02-10 at 10 54 55" src="https://user-images.githubusercontent.com/158411/218061400-f70b4e6f-3ca4-4338-a7b5-69de682722a4.png">

Reply:
okay yeah makes sense 👍